src/appMain/sdl_preloaded_pt.json

```diff
@@ -1,2563 +1,2571 @@
-                        },
-                        "pt-pt": {
-                            "tts": "%appName% está a solicitar as seguintes informações e permissões do veículo: %functionalGroupLabels%. Pode alterar estas permissões e ouvir descrições detalhadas no menu de definições das aplicações móveis. Prima \"\"Sim\"\" para permitir ou \"\"Não\"\" para recusar."
-                        },
-                        "ru-ru": {
-                            "tts": "%appName% запрашивает следующую информацию об автомобиле и разрешения: %functionalGroupLabels%. Вы можете изменить эти разрешения и прослушать подробные их описания в меню настроек мобильного приложения. Нажмите \"\"да\"\", чтобы предоставить разрешения, или \"\"нет\"\", чтобы не предоставлять."
-                        },
-                        "sv-se": {
-                            "tts": "%appName% begär tillgång till följande fordonsinformation och tillstånd: %functionalGroupLabels%. Du kan ändra tillstånden och höra detaljerade beskrivningar i menyn för mobilappsinställningar. Tryck Ja för att ge tillstånd eller Nej för att neka."
-                        },
-                        "tr-tr": {
-                            "tts": "%appName%, şu araç bilgilerini ve izinleri istiyor: %functionalGroupLabels%. Bu izinleri değiştirebilir ve mobil uygulamalar ayarlar menüsünden ayrıntılı açıklamaları dinleyebilirsiniz. Lütfen izin vermek için Evet'e veya reddetmek için Hayır'a basın."
-                        },
-                        "zh-cn": {
-                            "tts": "%appName% 正在请求下列车辆信息和权限： %functionalGroupLabels%。您可在移动应用程序设置菜单中更改这些权限，并听取详细说明。请按“是”允许权限或按“否”拒绝。"
-                        },
-                        "zh-tw": {
-                            "tts": "%appName% 正請求使用 %functionalGroupLabels% 的車輛資訊和許可。您可在行動應用程式設定清單中更改這些許可，並聆聽詳細說明。給予許可請按「是」，拒絕請按「否」。"
-                        }
-                    }
-                },
-                "AppPermissionsRevoked": {
-                    "languages": {
-                        "de-de": {
-                            "tts": "Die Autorisierungsdaten der App wurden geändert. %appName% hat keinen Zugriff auf %functionalGroupLabels% mehr. Installieren Sie die neueste Version der App auf Ihrem Gerät.."
-                        },
-                        "en-au": {
-                            "tts": "App authorizations have changed. %appName% can no longer access %functionalGroupLabels%. Please ensure you have the most recent app version installed on your mobile device."
-                        },
-                        "en-gb": {
-                            "tts": "App authorizations have changed. %appName% can no longer access %functionalGroupLabels%. Please ensure you have the most recent app version installed on your mobile device."
-                        },
-                        "en-ie": {
-                            "tts": "App authorizations have changed. %appName% ca
```

Comment:
fixed in 68de39168a4ca6a6e55a363d138a0147d0718906

Reply:
This was not fixed

Reply:
`OnServiceUpdate` notification was removed from `sdl_preloaded_pt.json` file in this commit

Reply:
To remove confusion, i've performed interactive rebase and removed `sdl_preloaded_pt.json` from commits, since the only change to this file were adding `OnServiceUpdate` notification and changed line endings.